report.tex

```diff
@@ -283,20 +283,16 @@ \subsubsection{Debugging}
 
 \subsection{Avatar Builder Issues}
 \subsubsection{Avatar URL}
-Manual entry for avatar URL is confusing and lacks guidance on obtaining a URL.
+Manual entry for avatar URL is confusing and lacks guidance on obtaining a URL. Additionally, the avatar URL entry sometimes fails to appear for input.
```

Comment:
This sounds like a point form thing, it might be prudent to expand this to a more prose-like form.